Packet++/header/TcpReassembly.h

```diff
@@ -102,13 +102,13 @@ struct ConnectionData
 	 * Set startTime of Connection
 	 * @param[in] startTimeValue integer value
 	 */
-	void setStartTime(const timeval &startTimeValue) { startTime = startTimeValue; }
+	void setStartTime(const timespec &startTimeValue) { startTime = startTimeValue; }
 
 	/**
 	 * Set endTime of Connection
 	 * @param[in] endTimeValue integer value
 	 */
-	void setEndTime(const timeval &endTimeValue) { endTime = endTimeValue; }
+	void setEndTime(const timespec &endTimeValue) { endTime = endTimeValue; }
```

Comment:
Same here: we can probably keep those with `timeval` (that will update both the old and new properties), and add an override for both methods that accept `timespec`